Libs/RaptureCore/src/main/java/rapture/kernel/AdminApiImpl.java

```diff
@@ -453,14 +448,10 @@ public void emailUser(CallingContext context, String userName, String emailTempl
         RaptureUser user = getUser(context, userName);
         if (user != null) {
             templateValues.put("user", user);
-            Mailer.email(context, emailTemplate, templateValues);
-        } else {
-            throw RaptureExceptionFactory.create(HttpURLConnection.HTTP_BAD_REQUEST, adminMessageCatalog.getMessage("NoExistUser", userName)); //$NON-NLS-1$
         }
+        Mailer.email(context, emailTemplate, templateValues);
```

Comment:
1. what happens if user is null?
2. The class uses RaptureExceptionFactory alot so does it make sense to change it for this one case?